docs/scripts/buildApi.ts

```diff
@@ -156,7 +156,11 @@ async function annotateComponentDefinition(context: {
 
   const demos = uniqBy<ReactApi['pagesMarkdown'][0]>(
     api.pagesMarkdown.filter((page) => {
-      return page.components.includes(api.name);
+      return (
+        page.components.includes(api.name) ||
+        (api.name.endsWith('Unstyled') &&
+          page.components.includes(api.name.replace('Unstyled', '')))
+      );
```

Comment:
Would this help future readers?

```suggestion
      // Testing for Unstyled avoids the need to mention the unstyled components in the
      // `components` key of the markdown header YAML.
      return (
        page.components.includes(api.name) ||
        (api.name.endsWith('Unstyled') &&
          page.components.includes(api.name.replace('Unstyled', '')))
      );
```